common/ripple/userUtils.py

```diff
@@ -581,166 +452,95 @@ async def getGameRank(userID: int, gameMode: int, relax_ap: int) -> int:
         board = "autoboard"
 
     position = await glob.redis.zrevrank(
-        f"ripple:{board}:{gameModes.getGameModeForDB(gameMode)}",
-        userID,
+        f"ripple:{board}:{gameModes.getGameModeForDB(game_mode)}",
+        user_id,
     )
 
     return int(position) + 1 if position is not None else 0
 
 
-async def getFriendList(userID: int):
-    """
-    Get `userID`'s friendlist
-
-    :param userID: user id
-    :return: list with friends userIDs. [0] if no friends.
-    """
-
-    # Get friends from db
-    # TODO: tuple cursor support? or use cmyui.mysql sync ver/make this native async
-    friends = await glob.db.fetchAll(
+async def get_friend_user_ids(user_id: int) -> list[int]:
+    """Get a user's friendlist."""
+    recs = await glob.db.fetchAll(
         "SELECT user2 FROM users_relationships WHERE user1 = %s",
-        [userID],
+        [user_id],
     )
-
-    if not friends or not len(friends):
-        # We have no friends, return 0 list
-        return [0]

```

Comment:
Dunno why this [0] case existed -- but I've removed support for it.

I've also made it so the login handler will only send the friendlist packet if `bool(await get_friend_user_ids(user_id))`.